src/TokenizerFwf.cpp

```diff
@@ -119,6 +132,19 @@ Token TokenizerFwf::nextToken() {
   if (!moreTokens_)
     return Token(TOKEN_EOF, 0, 0);
 
+  // Check for comments only at start of line
+  if (col_ == 0 && isComment(cur_)) {
```

Comment:
I think this will work in 99% of cases, but will fail rarely because of the crappy design of the whole parser - I think there are two other places where you need to do the same check. I'd say to not worry about that for now, as it's going to be rare, and will be more easily fixed as part of a bigger refactoring (#472)